templates/components/citation-style.blade.php

```diff
@@ -0,0 +1,21 @@
+<li>
```

Comment:
A couple of changes to make here to the Blade templates:
- We should be using standard code headers like you see in the `.tpl` files (and `.php` for that matter)
- Blade templates should use `.blade`, not `.blade.php`

Reply:
Now we support both the `.blade.php` and `.blade` so both can works . it's just that having only `.blade` make the code editor not to recognise the file properly with no code syntax highlight which make it hard to work with . I think there is probably workaround for code editor as well . updated it anyway .